wbemcli.py

```diff
@@ -191,29 +203,30 @@ def _remote_connection(server, opts, argparser_):
             stats_enabled=opts.statistics)
 
         try:
-            build_mock_repository(CONN, opts.mock_server, opts.verbose)
+            _build_mock_repository(CONN, opts.mock_server, opts.verbose)
         except ValueError as ve:
-            argparser_.error('Build Repository failed: %s' % ve)
+            _error("Building the mock repository failed: %s" % ve)
```

Comment:
Why not use argparser_.error here also. I must be missing something.

Reply:
`argparser.error()` displays a usage line because it reports command line errors. The ones I used `_error()` on are not argument errors, so I did not want to have the usage line.